dominos/src/main/java/me/fontys/semester4/dominos/configuration/data/store/StoreImporter.java

```diff
@@ -0,0 +1,108 @@
+package me.fontys.semester4.dominos.configuration.data.store;
+
+import me.fontys.semester4.data.entity.Store;
+import me.fontys.semester4.data.repository.StoreRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.*;
+
+@Configuration
+public class StoreImporter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StoreImporter.class);
+
+
+    @Qualifier("stores")
+    private final Resource[] stores;
+    private final StoreRepository storeRepository;
+    private final Map<String, Integer> warnings = new HashMap<>();
+
+    @Autowired
+    public StoreImporter(
+            Resource[] stores,
```

Comment:
This can easily be on a single line to reduce the lines of code, but is mostly a preference thing